app/assets/stylesheets/views/article-form.scss

```diff
@@ -0,0 +1,186 @@
+      );
+    }
+  }
+
+  &__logo {
+    display: flex;
+    margin-right: var(--layout-gap);
+
+    .logo {
+      width: 50px;
+      height: 40px;
+      padding: 0 7px 0 9px; // TODO: we have to fix SVG for logo a little...
+      background: var(--logo-bg);
+      fill: var(--logo-color);
+      border-radius: $br-default;
+    }
+  }
+
+  &__close {
+    margin-left: $su-4;
+    @media (min-width: $breakpoint-l) {
+      position: absolute;
+      right: $su-2;
+      top: $su-2;
+    }
+  }
+
+  &__tips {
+    position: sticky;
+    top: $su-9;
+  }
+
+  &__fields {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__body {
+    flex: 1 auto;
+    position: relative;
+    outline: none;
+  }
+
+  &__toolbar {
+    padding: $su-4 0;
+    position: sticky;
+    top: 0;
+    background: var(--card-bg);
+    transition: opacity $transition-props;
+
+    > :first-child {
+      margin-left: -$su-2;
+    }
+  }
+
+  &__cover {
+    margin-bottom: $su-6;
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+
+    @media (min-width: $breakpoint-s) {
+      flex-direction: row;
+      align-items: center;
+    }
+
+    &__image {
+      width: 250px;
+      height: 105px;
+      border-radius: $br-default;
+      object-fit: cover;
+      margin-bottom: $su-2;
+
+      @media (min-width: $breakpoint-s) {
+        margin-bottom: 0;
+        margin-right: $su-4;
+      }
+    }
+  }
+
+  &__title {
+    margin-bottom: $su-2;
+    position: relative;
+  }
+
+  &__tagsfield {
+    position: relative;
+  }
+
+  &__actions {
+    background: var(--base-10);
+    position: sticky;
+    bottom: 0;
+    padding: $su-4 $su-2;
+
+    @media (min-width: $breakpoint-l) {
+      padding: $su-6 $su-2;
+      padding-left: calc(
+        var(--layout-narrow-sidebar-width) + var(--layout-gap)
+      );
+    }
+
+    &__buttons {
+      display: flex;
+      align-items: center;
+    }
+  }
+}
+
+.crayons-article-form {
+  &__tagsoptions {
+    @extend .crayons-box;
+    @extend .crayons-box--level-2;
+
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: calc(100% + #{$su-4});
+    z-index: 100;
+    padding: $su-1;
+
+    @media (min-width: $breakpoint-s) {
+      left: -$su-2;
+      right: -$su-2;
+    }
+
+    @media (min-width: $breakpoint-m) {
+      left: -$su-4;
+      right: -$su-4;
+    }
+  }
+
+  &__tagname {
+    font-family: $ff-accent;
+    color: var(--base-90);
+    letter-spacing: -0.05em;
+    font-weight: bold;
+  }
+
+  &__tagoptionrow {
+    cursor: pointer;
+    color: var(--link-color);
+    padding: $su-3;
+
+    &:hover {
+      background: var(--link-bg-hover);
+      .crayons-tag {
+        color: var(--link-color-hover);
+      }
+    }
+
+    &--active {
+      background: var(--link-bg-hover);
+      .crayons-article-form__tagname {
+        color: var(--link-color-hover);
+      }
+    }
+  }
+
+  &__tagsoptionrulesbutton {
```

Comment:
another question for my curious mind, do we not use dashes or something similar with our crayons classnames, as a delimiter for words

Reply:
We do - this particular class though is not coming from crayons component BUT from one of our Preact components (but not crayons component): https://github.com/thepracticaldev/dev.to/blob/master/app/javascript/shared/components/tags.jsx - eventually we will probably refactor that to be crayonsified but that's just out of scope for this PR.